docs/rst/user_manual/configuration.rst

```diff
@@ -106,104 +106,184 @@ See `Interface Whitelist <https://fast-dds.docs.eprosima.com/en/latest/fastdds/t
+Its value should be decreased when the sample size and/or number of created endpoints (increasing with the number of topics) are big enough to cause memory exhaustion issues.
+If enough memory is available, however, the ``history-depth`` could be increased to deliver a greater number of samples to late joiners.
+
+.. _user_manual_configuration_dds__max_rx_rate:
+
+Max Reception Rate
+^^^^^^^^^^^^^^^^^^
+
+The ``max-rx-rate`` tag limits the frequency [Hz] at which samples are processed by discarding messages received before :code:`1/max-rx-rate` seconds have passed since the last processed message.
+It only accepts non-negative numbers.
+By default it is set to ``0``; it processes samples at an unlimited reception rate.
 
-    If no ``allowlist`` is provided, data will be read for all topics (unless filtered out in ``blocklist``).
+.. _user_manual_configuration_dds__downsampling:
 
+Downsampling
+^^^^^^^^^^^^
+
+The ``downsampling`` tag reduces the sampling rate of the received data by only keeping *1* out of every *n* samples received (per topic), where *n* is the value specified under the ``downsampling`` tag.
+When the ``max-rx-rate`` tag is also set, down-sampling only applies to messages that have passed the ``max-rx-rate`` filter.
+It only accepts positive integers.
+By default it is set to ``1``; it accepts every message.
+
+.. _user_manual_configuration_dds__manual_topics:
 
-Block topic list (``blocklist``)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-This is the list of topics that the |spy| will block, that is, all data published under the topics matching the
-filters specified in the ``blocklist`` will be discarded by the |spy| and therefore will not be relayed.
+Manual Topics
+-------------
 
-This list takes precedence over the ``allowlist``.
-If a topic matches an expression both in the ``allowlist`` and in the ``blocklist``, the ``blocklist`` takes precedence,
-causing the data under this topic to be discarded.
+A subset of :ref:`Topic QoS <user_manual_configuration_dds__topic_qos>` can be manually configured for a specific topic under the tag ``topics``.
+The tag ``topics`` has a required ``name`` tag that accepts wildcard characters.
+It also has two optional tags: a ``type`` tag that accepts wildcard characters and a ``qos`` tag with the :ref:`Topic QoS <user_manual_configuration_dds__topic_qos>` that the user wants to manually configure.
+If a ``qos`` is not manually configured, it will get its value by discovery.
+
+.. code-block:: yaml
+
+    topics:
+      - name: temperature/*
+        type: temperature/types/*
+        qos:
+          max-tx-rate: 15
+          downsampling: 2
+        participants:
+          - Participant0
+          - Participant1
```

Comment:
Add note:
`The Topic QoS configured in the Manual Topics take precedence over the Specs Topic QoS`.